src/io/flutter/sdk/FlutterSettingsConfigurable.java

```diff
@@ -108,27 +83,23 @@ public String getId() {
 
   @Nullable
   @Override
-  public Runnable enableSearch(String option) {
-    //TODO(pq): enable search
```

Comment:
No.  I think that's fine.  I didn't know about the automatic behavior and just didn't want to forget.  Let's remove it. 👍